Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM robopatrol:robopatrol-base
```

Comment:
`FROM robopatrol/robopatrol-base:latest`

Otherwise the container can not be found and pulled.